int32 id
int32 medicine_id
string description
string ingredient
string category
string usage_method
float64 unit_dose
string side_effects
string storage_conditions
string expiry_date
string barcode
string appearance_type
string notes